README.md

```diff
@@ -1 +1,75 @@
-# Math-Magician-React
\ No newline at end of file
+# Math Magician (React App)
```

Comment:
- [ ] Kindly delete the following boilerplate files as highlighted.

  ![wue](https://user-images.githubusercontent.com/36542986/198238804-0c35c8aa-8a06-4491-b40a-00b4fd3dadcf.jpg)